### ISSUE TITLE
dataset-description 

### ISSUE THREAD

**Issue by merbenf** [wosdetc/challenge#11]
hello safeshore team we are working on this dataset and we need some information and dataset description (it content ...) to add it in our research thesis  so can you please provide it

**Comment by dimouan** [comment]
Thank you for your interest in our dataset. You can find a description of the dataset in our paper lined below.  Please make sure you cite it for the usage of the dataset.

https://doi.org/10.1109/OJSP.2024.3379073

**Comment by merbenf** [comment]
yes sure , i already did sir i just want to know if i can upload this
dataset on roboflow to use it in my model please for my final year project ?
best regards .

**Comment by merbenf** [comment]
hello Wosdetc/Challenge team
i just wabt explain why i need to upload it on robofolow because i'm facing
issues when training the model on the dataset and the deadline is so near
so i need to upload it to roboflow and download it with pascalvoc format so
i can use it .
best regards
